src/scenario/scenario.cpp

```diff
@@ -605,3 +605,11 @@ std::string cScenario::get_feature_flag(std::string flag) {
 	if(iter == this->feature_flags.end()) return "";
 	return iter->second;
 }
+
+std::string cScenario::get_sdf_name(int row, int col) {
```

Comment:
Hmm… could also add a bounds check here, should we?

Reply:
Sure, I've done that.